linera-service/src/cli/command.rs

```diff
@@ -23,7 +23,7 @@ use linera_rpc::config::CrossChainConfig;
 use serde::Serialize;
 
 #[cfg(feature = "benchmark")]
-const DEFAULT_NUM_CHAINS: usize = 10;
+const DEFAULT_NUM_CHAINS_PER_CHAIN_GROUP: usize = 2;
```

Comment:
Yes… I was wondering if there is any reason to change this to something other than 2 at all.